pymc3/distributions/dist_math.py

```diff
@@ -30,11 +30,11 @@
 from aesara.scalar import UnaryScalarOp, upgrade_to_float_no_complex
 from aesara.scan import until
 from aesara.tensor.elemwise import Elemwise
+from aesara.tensor.math import gammaln
```

Comment:
I don't think we need to import this separately anymore. It's enough to use `at.gammaln` directly in the expression inside `multigammaln`

Reply:
For the `distributions/multivariate.py` where it is used a lot you can have a `from aesara.tensor import gammaln` directly there.